roles/resize_lv/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+# vim: set ft=ansible:
+#
+# This is a role that can be used to resize a logical volume on the host.
+# It requires that you know the name of the logical volume and will try
+# to determine the name of the volume group.  Since this is to be used on
+# Atomic Host systems, the logical volume names are well known, but the
+# volume group names can differ based on install method.
+#
+# Additionally, you must specify the desired size of the logical volume
+# in units of G.  If the current size of the logical volume is less than
+# the desired value, the volume will be extended.
+#
+# Required variables:
+#   - rl_lvname:  name of the logical volume
+#   - rl_lvsize:  desired size in G of the logical volume
+
+- name: Fail if variables aren't set
+  when: rl_lvname is not defined or
+        rl_lvsize is not defined
+  fail:
+    msg: "The variables 'rl_lvaname' and 'rl_lvsize' must be defined"
+
+- name: Determine the name of the volume group
+  command: vgs --no-headings -o vg_name
+  register: rl_vgs_name
+
+- name: Set vgname fact
+  set_fact:
+    rl_vgname: "{{ rl_vgs_name.stdout|trim }}"
+
+- name: Get current size of logical volume
+  command: "lvs -o lv_size --no-headings --units G --nosuffix {{ rl_vgname }}/{{ rl_lvname }}"
+  register: rl_lvs_size
+
+- name: Set current lvsize fact
+  set_fact:
+    rl_lvsize_current: "{{ rl_lvs_size.stdout|trim|round|int }}"
```

Comment:
I also had to update this one to `rl_lvsize_current: "{{ rl_lvs_size.stdout|trim|float|round|int }}"`